readthedocs/templates/profiles/private/delete_account.html

```diff
@@ -9,12 +10,31 @@
 {% block edit_content_header %} {% trans "Delete Account" %} {% endblock %}
 
 {% block edit_content %}
+
+  {% block delete-warning %}
+    <p>
+      <p>
+        <strong>All projects where you are the only owner will be deleted.</strong>
```

Comment:
Don't we have a red warning banner that we can use for this?

Reply:
We only have a "normal" banner, not a warning banner